docs/conf_sso_test_msscolab.rst

```diff
@@ -14,63 +14,72 @@ To set up a local identity provider with the mscolab server, you'll first need t
+To enable SAML2-based login (identity provider-based login), 
 
-After setting the `USE_SAML2`, the next step is to add the `CONFIGURED_IDPS` dictionary. This dictionary should include keys for each enabled Identity Provider, represented by `idp_identity_name`, and their corresponding `idp_name`. Once this dictionary is set up, it should be used to update various functionalities of the mscolab server, such as the SAML2Client config .yml file, ensuring proper integration with the enabled IDPs.
+- To start the process update `USE_SAML2 = True` in your `mscolab_settings.py` file.
 
+.. note::
+    After enabling the `USE_SAML2` option, the subsequent step involves adding the `CONFIGURED_IDPS` dictionary for the MSS Colab Server. This dictionary must contain keys for each active Identity Provider, denoted by their `idp_identity_name`, along with their respective `idp_name`. Once this dictionary is configured, it should be utilized to update several aspects of the mscolab server, including the SAML2Client configuration in the .yml file. This ensures seamless integration with the enabled IDPs. By default, configuration has been set up for the localhost IDP, and any additional configurations required should be performed by the developer.
 
-Generate metadata files
------------------------
+4. Generate metadata files
+--------------------------
 
-This involves generating necessary metadata files for both the identity provider and the service provider. You can generate them by simply running the appropriate command.
+This involves generating necessary metadata files for both the identity provider and the service provider. You can generate them by simply running the below command.
 
-Before executing this, you should set `USE_SAML2=True` as described in the third step(USE_SAML2).
+.. note::
+    Before executing this, you should set `USE_SAML2=True` as described in the third step(Enable USE_SAML2).
 
 `$ python mslib/mscolab/mscolab.py sso_conf --init_sso_metadata`
 
 
-Start Identity provider
------------------------
+5. Start Identity provider
+--------------------------
 
-Once you setted certificates and metada files you can start mscolab server and local identity provider. To start local identity provider, simpy execute
+Once you set certificates and metada files you can start mscolab server and local identity provider. To start local identity provider, simply execute:
 
-`$ python mslib/idp/idp.py idp_conf`
+`$ msidp`
 
 
-Start the mscolab Server
-------------------------
+6. Start the mscolab Server
+---------------------------
 
 Before Starting the mscolab server, make sure to do necessary database migrations.
 
-When this is the first time you setup a mscolab server, you have to initialize the database by
+When this is the first time you setup a mscolab server, you have to initialize the database by:
 
 `$ python mslib/mscolab/mscolab.py db --init`
 
```

Comment:
the documentation is always meant for the Users, so they use only entrypoints.

Only the section Development ist for us :)